src/service/donation.ts

```diff
@@ -30,6 +31,16 @@ export function useCreateDonation() {
   }
 }
 
+export function useCreateBankDonation() {
+  // const { keycloak } = useKeycloak<KeycloakInstance>()
+  return async (data: DonationBankInput) => {
+    return await apiClient.post<DonationResponse, AxiosResponse<DonationResponse>>(
+      endpoints.donation.createBankDonation.url,
+      data,
+    )
+  }
+}
+
```

Comment:
This endpoint is only for administrators and regular users will receive `401 Forbidden` when they try to donate